docs/how-to-setup-wsl.md

```diff
@@ -0,0 +1,218 @@
+# Setting up WSL for Working with freeCodeCamp Locally
+
+Follow these guidlines for setting up WSL to set up freeCodeCamp locally.
+
+This guide is covers some common issues encountered with the setup of WSL.
+
+## Enable WSL & Download a Distro
+
+1) Follow the instructions given in the documentation by Microsoft to install WSL2 (Recommended Distro: Ubuntu18.04):
+[Installing WSL2](https://docs.microsoft.com/en-us/windows/nodejs/setup-on-wsl2#install-windows-10-insider-preview-build)
+
+> [!NOTE]
+> You can download a different distro, but the setup might differ.
+
+2) On the same page, ensure to follow the instructions about installing Nodejs and NPM through the NVM.
+
+**NOTE:** If you encounter errors on step 2, skip to the next step, and return when prompted
+
+## Setting Up Git
+
+It is likely you will be unable to clone your fork of freeCodeCamp, without running these steps, but there is no harm in trying. Just follow the instructions on [the Contributors Site](https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally).
+
+If you run into any issues, follow through with one alternative in this step:
```

Comment:
This suggests that any of the alternative is valid and should work, but it looks like they need to try them in an order?

Reply:
I will reword this. I suppose their is an order of hopefulness. It is not structured in an order I followed, but I just come into a lot of errors.

The alternatives are there as a hopeful way to fix any issues pertaining to Git. The order is based on length.